boards/phytec/phyboard_polis/doc/index.rst

```diff
@@ -1,4 +1,4 @@
-.. _mimx8mm_phyboard_polis:
+.. _phyboard_polis:
 
 PhyBOARD Polis (NXP i.MX8M Mini)
```

Comment:
Can we fix both titles in this PR as well?

Should be called `phyBOARD-Polis (NXP i.MX8M Mini)` or `phyBOARD-Polis i.MX8M Mini`

Reply:
Just fixed all occasions of the name and corrected it for both boards. Should now be aligned over all PHYTEC boards.